go/multiorch/coordinator/leader_appointment.go

```diff
@@ -24,15 +25,15 @@ import (
 	multipoolermanagerdatapb "github.com/multigres/multigres/go/pb/multipoolermanagerdata"
 )
 
-// BeginTerm implements stages 1-4 of the consensus protocol:
+// BeginTerm implements stages 1-5 of the consensus protocol:
 // 1. Discover max term from all nodes
 // 2. Increment to get new term
 // 3. Select candidate based on WAL position
 // 4. Send BeginTerm RPC to all nodes in parallel
-// 5. Validate quorum (majority acceptance)
+// 5. Validate quorum using durability policy
 //
 // Returns the candidate node, standbys, the new term, and any error.
-func (c *Coordinator) BeginTerm(ctx context.Context, shardID string, cohort []*Node) (*Node, []*Node, int64, error) {
+func (c *Coordinator) BeginTerm(ctx context.Context, shardID string, cohort []*Node, quorumRule *clustermetadatapb.QuorumRule) (*Node, []*Node, int64, error) {
 	// Stage 1: Obtain Term Number - Query all nodes for max term
 	c.logger.InfoContext(ctx, "Discovering max term", "shard", shardID, "cohort_size", len(cohort))
 	maxTerm, err := c.discoverMaxTerm(ctx, cohort)
```

Comment:
Just for my memory. My recollection is that we aligned on not discoveringMaxTerm in this way. 

Is that still the plan?

Reply:
That is correct. Another piece of data that multiorch will use its memory for. This is temporary until then.